### PR TITLE
renoise: Add desktop item

### PR DESCRIPTION
**Pull request by turboMaCk**
Adds desktop item to renoise.

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/audio/renoise/default.nix [hunk 1]
```diff
@@ -47,6 +47,18 @@ stdenv.mkDerivation rec {
...
     ln -s $out/renoise $out/bin/renoise
+
+    // Desktop item
+    mkdir -p $out/share/applications
+    cp -r Installer/renoise.desktop $out/share/applications/renoise.desktop
+
+    // Desktop item icons
+    mkdir -p $out/share/icons/hicolor/48x48/apps
+    mkdir -p $out/share/icons/hicolor/64x64/apps
+    mkdir -p $out/share/icons/hicolor/128x128/apps
```
Comment (SuperSandro2000) [inline pkgs/applications/audio/renoise/default.nix:58]:
```suggestion
    mkdir -p $out/share/icons/hicolor/{48x48,64x64,128x128}/apps
```

Comment (turboMaCk) [reply pkgs/applications/audio/renoise/default.nix]: ah good idea, thanks

**Comment by turboMaCk** [comment]
not a super high on a priority list but I think this can be merged. I'm using overlay with this for last few days and it improves integration to desktops a lot. cc @SuperSandro2000

**Review by rmcgibbo — APPROVED** [review]

File: pkgs/applications/audio/renoise/default.nix [hunk 2]
```diff
@@ -47,6 +47,16 @@ stdenv.mkDerivation rec {
...
+
+    # Desktop item
+    mkdir -p $out/share/applications
+    cp -r Installer/renoise.desktop $out/share/applications/renoise.desktop
+
+    # Desktop item icons
+    mkdir -p $out/share/icons/hicolor/{48x48,64x64,128x128}/apps
+    cp Installer/renoise-48.png $out/share/icons/hicolor/48x48/apps/renoise.png
+    cp Installer/renoise-64.png $out/share/icons/hicolor/64x64/apps/renoise.png
+    cp Installer/renoise-128.png $out/share/icons/hicolor/128x128/apps/renoise.png
```
Comment (SuperSandro2000) [inline pkgs/applications/audio/renoise/default.nix:59]:
```suggestion
    cp Installer/renoise-{48,64,128}.png $out/share/icons/hicolor/{48x48,64x64,128x128}/apps/renoise.png
```

**Pull request merged**